Library/Homebrew/cmd/update.sh

```diff
@@ -810,6 +810,8 @@ EOS
       else
         echo "Failed to download ${json_url}!" >>"${update_failed_file}"
       fi
+
+      rm -f "${HOMEBREW_CACHE}/api/${formula_or_cask}.json"
```

Comment:
Maybe mention in a comment this is the legacy file so isn't mistaken as a typo or something.